cmd/vic-machine/create/create_test.go

```diff
@@ -260,3 +260,41 @@ func TestParseGatewaySpec(t *testing.T) {
 		}
 	}
 }
+
+func TestProcessVolumeStoreParam(t *testing.T) {
+	posiitveTestCases := []string{
```

Comment:
positiveTestCases?  looks like you somehow managed to mispell it the exact same way below as well :)